### ISSUE TITLE
Color Accessibility threshold

### ISSUE THREAD

**Issue by oliviertassinari** [mui-org/material-ui-x#2]
### Potential issue

<img width="404" alt="Capture d’écran 2020-05-05 à 18 29 56" src="https://user-images.githubusercontent.com/3165635/81166393-e600d480-8f93-11ea-9696-b8b73699dd0d.png">

https://muix-storybook.netlify.app/?path=/story/real-data-demo--commodity

I see two issues in the above case:

1. The color contrast of the checkbox is **1.39**:1 https://webaim.org/resources/contrastchecker/?fcolor=F50057&bcolor=4A98EC. It fails to match the WCAG AA minimum accessibility target (**4.5**:1).
2. Luckily, Google has us covered :slightly_smiling_face: https://material.io/design/interaction/states.html#selected

### Potential solution

I think that we can solve the above problems with:

1. Use the primary color for the Checkbox
2. Patch the selected background color until we update the value to match the specification in the core. 8% of the opacity of the primary in light mode and 16% in dark mode.
